examples/jsm/shaders/SMAAShader.d.ts

```diff
@@ -30,13 +30,3 @@ export interface SMAAWeightsShader {
   vertexShader: string;
   fragmentShader: string;
 }
-
-export interface SMAAWeightsShader {
```

Comment:
This should be `export interface SMAABlendShader`. Sorry, for the copy/paste error.

Reply:
Restored with proper name.